### ISSUE TITLE
Detection Language Issue preventing Device Detection

### ISSUE THREAD

**Issue by djskripta** [HandsetDetection/php-apikit#44]
Hi,

We are getting errors like `*Undefined index: en_us*​ ​_HDExtra.php@117_​` after deploying code using this package. This prevents scripts from completing device detection and results in a fatal error.

Locally, and on our beta server, we never saw this error.
Not sure if this is a permissions issue or something else.

Looping in @artkay

**Comment by artkay** [comment]
A little addition - probably we never saw that b/c we never had such a big traffic on the staging servers.

**Comment by artkay** [comment]
Few more records from our logs with more languages that are triggering the error (could be useful for testing):

```
*Undefined index: nl-nl*​ ​_HDExtra.php@117_​
*Undefined index: es-xl*​ ​_HDExtra.php@117_
*Undefined index: ru-ru*​ ​_HDExtra.php@117_​
*Undefined index: tr-tr*​ ​_HDExtra.php@117_
*Undefined index: en-*​ ​_HDExtra.php@117_​
*Undefined index: sv-se*​ ​_HDExtra.php@117_
*Undefined index: en_gb*​ ​_HDExtra.php@117_​
```

**Comment by elricho** [comment]
Thanks for the patch. Its a good defensive suggestion for unknown languages. I've merged #44. 👍
